### ISSUE TITLE
Embedding concatenation mismatch between paper and code: is Bayesian-sampled sequencing depth additionally concatenated?

### ISSUE THREAD

**Issue by Bioshixin** [biomed-AI/CellFM-torch#1]
Hello, and thank you for releasing the code and this excellent work.

I noticed a possible discrepancy between the paper and the implementation regarding how input embeddings are concatenated:

Paper: inputs are described as a concatenation of [CLS] and gene embeddings, without a sequencing-depth feature.

Code: the embedding appears to additionally concatenate a sequencing-depth feature derived from the Bayesian sampling step.

Could you please clarify:

- Is this extra depth feature an intentional design update, or is it required to leverage the previously Bayesian-sampled data?
- 
- If intentional, what is the rationale, and should this feature be enabled to reproduce the results reported in the paper? What configuration do you recommend for reproduction?
- 
- If not intentional, what is the recommended fix or configuration?

For your reference, here are some pointers from my setup (if helpful):

**Comment by Bioshixin** [comment]
expr_emb = torch.cat([cls_token, st_emb, expr_emb], dim=1)
